lua/extn/sky/unstable/io/arc.lua

```diff
@@ -72,36 +163,53 @@ end
 --
 local ArcDialRender = sky.Object:extend()
 
-TWO_PI = math.pi * 2
-SLIM_WIDTH = TWO_PI / 64
+local TWO_PI = math.pi * 2
+local SLIM_WIDTH = TWO_PI / 64
 
 function ArcDialRender:new(props)
   ArcDialRender.super.new(self, props)
   self.which = props.which or 1
   self.where = props.where or props.which
   self.width = props.width or SLIM_WIDTH
   self.level = props.level or 8
+  if props.target_indicator ~= nil then self.target_indicator = props.target_indicator
+  else self.target_indicator = true end
+  self.target_level = props.target_level or props.level or 15
   self.mode = props.mode or 'pointer'
 end
 
 function ArcDialRender:clear(props, which)
   props.arc:segment(which, 0, TWO_PI, 0)
 end
 
+local function round(x)
```

Comment:
Different from util.round? An optimization maybe?

Reply:
weirdly i missed that `util.round` was a thing. looking at the `util.round` implementation i'm leaning towards keeping this simple version of round since it is minimal and sufficient for the render logic and not being exported.